lua/bufferline/bbye.lua

```diff
@@ -101,7 +101,7 @@ function bbye.delete(action, force, buffer, mods)
     return
   end
 
-  vim.w.bbye_back = true
+  local current_window = vim.api.nvim_get_current_win()
```

Comment:
The original `bbye` needed this hack because vim didn't have stable window IDs, it only had window numbers.